scene/resources/dynamic_font.cpp

```diff
@@ -1083,8 +1083,21 @@ void DynamicFont::update_oversampling() {
 				E->self()->outline_data_at_size->update_oversampling();
 			}
 
+			for (int i = 0; i < E->self()->fallback_data_at_size.size(); i++) {
+				if (E->self()->fallback_data_at_size[i].is_valid()) {
+					DynamicFontAtSize *font = const_cast<DynamicFontAtSize *>(E->self()->fallback_data_at_size[i].ptr());
```

Comment:
Is the cast necessary? I think you could use `E->self()->fallback_data_at_size.write[i]->update_oversapling()` instead.